models/issue_list.go

```diff
@@ -154,38 +154,39 @@ func (issues IssueList) loadMilestones(e Engine) error {
 	return nil
 }
 
-func (issues IssueList) getAssigneeIDs() []int64 {
-	var ids = make(map[int64]struct{}, len(issues))
-	for _, issue := range issues {
-		if _, ok := ids[issue.AssigneeID]; !ok {
-			ids[issue.AssigneeID] = struct{}{}
-		}
-	}
-	return keysInt64(ids)
-}
-
 func (issues IssueList) loadAssignees(e Engine) error {
-	assigneeIDs := issues.getAssigneeIDs()
-	if len(assigneeIDs) == 0 {
+	if len(issues) == 0 {
 		return nil
 	}
 
-	assigneeMaps := make(map[int64]*User, len(assigneeIDs))
-	err := e.
-		In("id", assigneeIDs).
-		Find(&assigneeMaps)
+	type AssigneeIssue struct {
+		Assignee      *User           `xorm:"extends"`
+		IssueAssignee *IssueAssignees `xorm:"extends"`
+	}
+
+	var assignees = make(map[int64][]*User, len(issues))
+	rows, err := e.Table("issue_assignees").
+		Join("INNER", "issue", "`issue`.id = `issue_assignees`.issue_id").
```

Comment:
Join issue is unnecessary. And for keep the sequence with `AssigneeIssue`, maybe add 
`Select("user.*, issue_assignees.*")`
or change `AssigneeIssue` struct to 
```Go
type AssigneeIssue struct {
    IssueAssignee *IssueAssignees `xorm:"extends"`
    Assignee      *User           `xorm:"extends"`
}
```